### PR TITLE
Create unprepared denials report

### PR DESCRIPTION
**Pull request by cmgiven**
Connects #2887

### CODE REVIEW

File: app/services/unprepared_denials_report.rb [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+  def load_record(case_record)
+    AppealRepository.build_appeal(case_record)
+  end
+
+  def include_record?(appeal)
+    appeal.decisions.empty?
+  end
+
+  def cleanup_record(appeal)
+    appeal.clear_documents!
```
Comment (sharonwarner) [inline app/services/unprepared_denials_report.rb]: Why is this necessary?

Comment (cmgiven) [inline app/services/unprepared_denials_report.rb]: When we first started writing these reports way back, this was included to keep memory usage in check. Possibly no longer needed now that we have Postgres?
